tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/date_and_time.py

```diff
@@ -21,3 +29,59 @@ def __init__(self, marionette):
     def select_time_format(self, time_format):
         self.marionette.find_element(*self._24h_selector_locator).tap()
         self.select(time_format)
+
+    def set_automatic_time_update(self, state):
+        """
+        toggles 'Set Time Automatically' flag
+        """
+
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(
+                expected.element_present(*self._autotime_enabled_locator))))
+
+        if not state:
+            if self.is_autotime_enabled is True:
+                self.marionette.find_element(*self._autotime_enabled_switch_locator).tap()
+            self.wait_for_condition(lambda m: self.is_autotime_enabled is False)
+        else:
+            if self.is_autotime_enabled is False:
+                self.marionette.find_element(*self._autotime_enabled_switch_locator).tap()
+            self.wait_for_condition(lambda m: self.is_autotime_enabled is True)
+
+    def set_region(self, region):
+        """
```

Comment:
I don't think this documentation adds much value to the name of the function. Looks like we can remove it to me.